### ISSUE TITLE
Improve color contrast

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/ph-scale#310]
To improve color contrast, the following components need color updates. I think it would be nice if we did this together @marlitas to ensure that the new colors look good. 

- [ ] pH probe color (and color of pH meter readout when probe is in solution)

   - change probe color to very dark green, hex value= 102D06, or black
   - change readout background to match? Let's see what that looks like.

- [ ] pH meter readout when probe is not in solution

<img width="123" alt="Image" src="https://github.com/user-attachments/assets/f35717b4-18e6-43ce-afb7-98bdd0eeadd6" />

   - make it darker gray, hex value 757575

- [ ] Text giving particle amounts 

<img width="175" alt="Image" src="https://github.com/user-attachments/assets/fc3557bc-e181-4838-964a-e442c3aad853" />

   - make text black

- [ ] Grayed out toggle switch text on gray and white backgrounds

<img width="309" alt="Image" src="https://github.com/user-attachments/assets/8ae96988-bd94-4cb1-84fb-ec2ed0a6af3c" />

<img width="226" alt="Image" src="https://github.com/user-attachments/assets/deba07a7-3e2b-4481-9b2f-c21efbb2e57f" />

   - Can grayed out text be darker to hex value 6E6E6E, AND make background of panel a lighter gray to hex value EDEDED?

**Comment by pixelzoom** [comment]
I added a few more ProfileColorProperty instances to PHScaleColors.ts in https://github.com/phetsims/ph-scale/commit/a2d4337fb23ea474acf2437cae1b9ad7f5784b31, while answering some historical questions from @arouinfar.

History of color profile is in https://github.com/phetsims/ph-scale/issues/254.

Not all of the colors are in PHScaleColors.ts.  Solute colors are in Solute, where static instances of `Solute` are defined. Each instance has 3 associated colors, see SoluteOptions.  Solute `computeColor` does the interpolation between these colors, based on the concentration.   Making this support ProfileColorProperty will not be easy, and will be difficult to manage in Color Editor.

**Comment by pixelzoom** [comment]
If you end up changing the ProfileColorProperty values, you may need to programmatically create the screen icons.  They should be programmatically created regardless, probably were not because they were ported from Java, or because this was one of the first HTML5 sims.  And there's certainly no need for separate artwork for home screen and navbar, as is the case now.

**Comment by arouinfar** [comment]
I'll leave the full review to @Nancy-Salpepi, but I had an idea for the My Solution screen.

> Done above, however the particle counts now overlap the probe in the My Solution screen since they are wider. How would you like me to resolve?

@marlitas could we try moving the probe to the left a bit? The AccordionBox could stay in the same spot, just shift the probe bit.

**Comment by marlitas** [comment]
I agree! The dark green probe wasn't my favorite. I like the purple it works nicely with the other colors in the sim :-)

Let me know what you think and feel free to close if all looks well!

**Comment by marlitas** [comment]
Oops missed @arouinfar's comment about moving the probe. Done above and this is ready for review!

**Comment by Nancy-Salpepi** [comment]
Both types of probes look good!
Thanks @marlitas!
